src/listeners/substrate-listener/commands/service-request/service-request-unstaked/service-request-unstaked.handler.ts

```diff
@@ -32,7 +36,7 @@ export class ServiceRequestUnstakedHandler
       const stakingLogging: TransactionLoggingDto = {
         address: serviceRequest.requester_address,
         amount: convertToDbioUnit(serviceRequest.staking_amount),
-        created_at: new Date(),
+        created_at: new Date(this.dateTimeProxy.now()),
```

Comment:
How about using `this.dateTimeProxy.new()`?

Reply:
Refer here: https://github.com/debionetwork/debio-backend/blob/8315801b8b1f5ce19fe5221995f5f95918172566/src/common/modules/proxies/date-time/date-time.proxy.ts#L6